src/components/Options/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+
+  font-size: 0.9em;
+  margin-top 20px;
+  margin-bottom 20px;
+  `
+
+export default class Options extends Component {
+
+  render(){
+    return (
+      <Wrapper>
+      <div>
+        Radio Button:
```

Comment:
All form elements should have an associated `label` either with `input` inside or using the `for` and `id` attributes to associate label with input. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label